mappings/net/minecraft/world/featureToggle/FeatureSet.mapping

```diff
@@ -0,0 +1,29 @@
+CLASS net/minecraft/class_7699 net/minecraft/world/featureToggle/FeatureSet
+	FIELD field_40172 MAX_FEATURE_FLAGS I
+	FIELD field_40173 EMPTY Lnet/minecraft/class_7699;
+	FIELD field_40174 branch Lnet/minecraft/class_7700;
+	FIELD field_40175 featuresMask J
+	METHOD <init> (Lnet/minecraft/class_7700;J)V
+		ARG 1 branch
+		ARG 2 featuresMask
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 featureSet
+	METHOD method_45397 empty ()Lnet/minecraft/class_7699;
+	METHOD method_45398 of (Lnet/minecraft/class_7696;)Lnet/minecraft/class_7699;
+		ARG 0 feature
+	METHOD method_45399 of (Lnet/minecraft/class_7696;[Lnet/minecraft/class_7696;)Lnet/minecraft/class_7699;
+		ARG 0 feature1
+		ARG 1 features
+	METHOD method_45400 isSubsetOf (Lnet/minecraft/class_7699;)Z
```

Comment:
`isCompatibleWith` (usage wise) or `areFlagsEqual`?

Reply:
I feel like `areFlagsEuqal` is not entirely accurate as if B contains elements not in A, the result is still true. `isCompatibleWith` doesn't describe what is actually does.

Reply:
hmm